src/Util/Escape.php

```diff
@@ -61,6 +61,9 @@ public static function shellArg($arg, $os = null)
      */
     public static function isWindows($os)
```

Comment:
Might as well provide a default value here so that folks do not need to explicitly specify null when calling.

Reply:
Done